src/index.js

```diff
@@ -87,7 +90,8 @@ class GifPlayerContainer extends React.Component {
 
 GifPlayerContainer.propTypes = {
   gif: PropTypes.string,
-  still: PropTypes.string
+  still: PropTypes.string,
+  autoPlay: PropTypes.bool,
```

Comment:
Can we change `autoPlay` to `autoplay`? Weird nitpick I know. 😉

Reply:
Originally I used `autoplay` and React yelled at me saying it does not know what to do and if I meant `autoPlay` instead

Reply:
Hmmm... your editor probably thinks you were trying to edit an `<audio>` element.